### DISCUSSION TITLE
Filter Contribution

### DISCUSSION THREAD

**Discussion by jithendra945** [cuebook/CueObserve#96] [Q&A] [answered]
Can you please explain, 
What is Filter's Contribution, and how it is calculated.

**Comment by sachinkbansal** [comment, marked as answer]
Say you have a dataset that has daily orders data for the last 12 months.
You have defined an anomaly on this dataset as: `Orders` `State Top 10`

Your total orders across all states were 1 million. California was your # 1 state with 100K orders during the 12 month period. 

Then `State = California` filter's contribution is 10% (100K/1M) to Orders

**Reply by jithendra945** [reply]
Oh. Got it. Thanks Sachin.
